portfolio/src/main/webapp/script.js

```diff
@@ -246,3 +246,29 @@ function filter(location) {
 
 // Append the "mapscript" element to "head"
 document.head.appendChild(mapscript);
+
+// Create GET request to /upload server and display file uploads
+function showFiles() {
+  console.log("showing files");
+  fetch("/upload").then(response => response.json()).then(files => {
+    // Add individual files to html page
+    const list = document.getElementById("uploads");
+    list.innerHTML = "";
+    files.forEach((file) => { 
+      const listElement = document.createElement("li");
+
+      // Display image, which always needs to be rotated 90 degrees for the correct orientation
```

Comment:
why is this the case? Like does the api not upload the images how the user uploads it?

Reply:
Initially, all the images I tested with would be rotated 90 degrees, so I thought it always did that. After testing with more images, I noticed that it only sometimes rotates them. I deleted the line of code that rotated the image and forgot to change the comment! But I'm not sure why some get rotated while others don't